docs/api.md

```diff
@@ -350,6 +350,66 @@ class ResetPasswordPage extends React.Component {
 }
 ```
 
+#### ChangePasswordForm
+
+Renders a change password form. The parameter `spToken` is required in order for the token to be validated.
+
+```html
+<ChangePasswordForm spToken={this.props.location.query.sptoken} />
+```
+
+Customize the form by providing your own markup.
+
+```html
+<ChangePasswordForm>
```

Comment:
I think it's bad practice to have code examples that doesn't work.

And to best show the difference between a default and a custom form, I think it would be best to not remove things but rather show what additional things would make it *custom* (see my previous comment).